hack/version.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```

Comment:
did kubernetes have any sorts of tests that came with this script we can use to verify behavior?

Reply:
I cant find any tests they use for this, but if there is a problem with this its will fail on the test I am writing now